NeuroID/DataStore.swift

```diff
@@ -3,6 +3,7 @@ import Foundation
 public enum DataStore {
     static var _events = [NIDEvent]()
     private static let lock = NSLock()
+    private static let max_event_size = 2000
```

Comment:
Wouldn't it be better to have the max size set to the same as android (1999) or vice versa so that we could extract it to a global config later?